account_invoice_view_adj/views/account_invoice_view_adj.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
+        <record id="view_account_invoice_supplier_tree_inherit" model="ir.ui.view">
+            <field name="name">account.invoice.supplier.tree.inherit</field>
+            <field name="model">account.invoice</field>
+            <field name="inherit_id" ref="account.invoice_supplier_tree"/>
+            <field name="arch" type="xml">
+                <xpath expr="//tree/field[@name='state']" position="before">
+                   <field name="user_id" string="Responsible"/>
+                </xpath>
+            </field>
+        </record>
+    </data>
```

Comment:
```suggestion
```

Reply:
Thank you!
I know the reason for removing </data>.
starting with Odoo version 11.0, the <data> tag is optional and can be omitted when writing module data files.